core/combinedvideosource.go

```diff
@@ -0,0 +1,283 @@
+package core
+
+import (
+	"sync"
+
+	"github.com/ericxtang/m3u8"
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/drivers"
+	"github.com/livepeer/go-livepeer/net"
+	ffmpeg "github.com/livepeer/lpms/ffmpeg"
+	"github.com/livepeer/lpms/stream"
+)
+
+var liveListLev uint = 10
+
+// CombinedVideoSource holds data in different OSes
```

Comment:
Can we expand on this?  For example, how `CombinedVideoSource` is intended to be used.